ci/jenkins/test.sh

```diff
@@ -201,7 +201,7 @@ function clean_antrea {
     for antrea_yml in ${WORKDIR}/*.yml; do
         kubectl delete -f $antrea_yml --ignore-not-found=true || true
     done
-    docker images --format "{{.Repository}}:{{.Tag}}" | grep 'antrea'| xargs -r docker rmi -f || true
+    docker images --format "{{.Repository}}:{{.Tag}}" | grep '${BUILD_TAG}' | xargs -r docker rmi || true
```

Comment:
maybe we could search for both `antrea` and the BUILD_TAG?

Reply:
but won't that work the same?

Reply:
yes, but it makes it more explicit that we are only deleting (a subset of) Antrea images